src/storage/store/csr_node_group.cpp

```diff
@@ -22,15 +22,15 @@ bool CSRNodeGroupScanState::tryScanCachedTuples(RelTableScanState& tableScanStat
     const auto startCSROffset = header->getStartCSROffset(boundNodeOffsetInGroup);
     const auto csrLength = header->getCSRLength(boundNodeOffsetInGroup);
     nextCachedRowToScan = std::max(nextCachedRowToScan, startCSROffset);
-    if (nextCachedRowToScan >= numScannedRows ||
-        nextCachedRowToScan < numScannedRows - numCachedRows) {
+    if (nextCachedRowToScan >= nextRowToScan ||
```

Comment:
Why are you doing this renaming in this PR? Can you revert? Or get @ray6080 to review this.

Reply:
This code is from @benjaminwinger